### PR TITLE
Feature/dockerfile

### PR DESCRIPTION
**Pull request by fgypas**

### CODE REVIEW

**Comment by agardelakos** [comment]
update README please

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+LABEL software.tags="workflows,monitor,track"
+LABEL maintainer="fgypas@gmail.com"
+LABEL maintainer.organisation=""
+LABEL maintainer.location=""
+LABEL maintainer.license="MIT"
+
+##### VARIABLES #####
+# Use variables for convenient updates/re-usability
+ENV FLASK_APP=server/app.py
+ENV FLASK_ENV=development
```
Comment (gkostoulas) [inline Dockerfile]: We may not use this

Comment (fgypas) [reply Dockerfile]: Done

**Comment by fgypas** [comment]
This fixes #6

**Pull request merged**